admin_manual/configuration/server/security/password_policy.rst

```diff
@@ -0,0 +1,61 @@
+   :alt: The Password Policy application
+
+.. caution:: The Security and Password Policy apps are mutually exclusive.
+
+From the 2.0.0 release of `the Password Policy app`_, all ownCloud users (both enterprise and community) have the option of installing and enabling the application.
+The Password Policy application enables administrators to define password requirements for user passwords and public links.
+
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Setting                                                                            | User Passwords | Public Link Passwords |
++====================================================================================+================+=======================+
+| Specify a valid password requirements.                                             | Yes            | Yes                   |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Specify a password expiration period.                                              | Yes            | Yes                   |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Specify expiration dates for public link shares.                                   | No             | Yes                   |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Forced password change on first login.                                             | Yes            | No                    |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Disallowing passwords that match a configurable number of previous passwords       | Yes            | Yes                   |
+| (defaults to the previous 3).                                                      |                |                       |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Users can be notified a configurable number of days before their password expires. | Yes            | No                    |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+| Users will be notified when their password has expired.                            | Yes            | No                    |
++------------------------------------------------------------------------------------+----------------+-----------------------+
+
+.. note::
+   Active user sessions will **not** end when passwords expire.
+   However, a password change will be forced when the user session expires (e.g., on logout).
+   Token access for app or client authentication is not affected.
+
+.. note::
```

Comment:
We still need a note here clearly stating that when imposing password expiration policies on instances that have existing users it's mandatory to run the occ command once (**after enabling the expiration policy in the web UI**) to set an initial password change date for all users that will be used by the policy in subsequent iterations. This information is somehow stated on the "occ commands" doc page so it could be mentioned as a short note and linked but I find it way more important to state this information on the actual password policy doc page as expiration policies simply won't work if this step is not taken into account.

Reply:
Will there be an `user:expire-password` switch that handles expiring passwords for all users simultaneously?